amaascore/transactions/position.py

```diff
@@ -44,4 +44,5 @@ def average_price(self, value):
         :param value:
         :return:
         """
-        self._average_price = Decimal(value)
\ No newline at end of file
+        if value is not None:
+            self._average_price = Decimal(value)
```

Comment:
Trying to read average value for an object with default average value will cause exception.. is it expected?

Reply:
Ahh good call, thanks.  I've updated the getter property